src/main/java/lotto/model/Rank.java

```diff
@@ -17,14 +18,27 @@ public enum Rank {
 		this.prize = prize;
 	}
 
-	public static Rank from(final int count) {
+	public static Rank match(final int count, final boolean matchBonus) {
+		if(SECOND.count == count){
+			return rankByBonus(matchBonus);
+		}
+
 		return Arrays.stream(values())
 			.filter(rank -> rank.count == count)
 			.findAny()
 			.orElse(MISS);
 	}
 
+	private static Rank rankByBonus(final boolean matchBonus) {
+		if(matchBonus){
```

Comment:
```suggestion
		if (matchBonus) {
```

이것 이외에도 컨벤션을 맞춰주시면 좋을 것 같아요 👍